lib/fencing/st_client.c

```diff
@@ -442,17 +442,22 @@ make_args(const char *action, const char *victim, uint32_t victim_nodeid, GHashT
             alias = g_hash_table_lookup(port_map, victim);
         }
 
-        /* Always supply the node's name too:
-         *    https://fedorahosted.org/cluster/wiki/FenceAgentAPI
-         */
-        append_const_arg("nodename", victim, &arg_list);
```

Comment:
Perhaps I misunderstood originally... can we not leave `make_args()` alone and just change fence_legacy to look for `nodename` instead of `port`?